spec/controllers/users/reset_passwords_controller_spec.rb

```diff
@@ -181,7 +181,9 @@
         raw_reset_token, db_confirmation_token =
           Devise.token_generator.generate(User, :reset_password_token)
 
-        Timecop.freeze(Time.zone.now) do
+        now = Time.zone.now
+        freeze_time do
+          travel_to(now)
```

Comment:
Could this be ... ?

```suggestion
        travel_to Time.zone.now do
```

Or even ... ?

```suggestion
        freeze_time do
```

Reply:
It doesn't seem like `now` is ever used, so I took the latter.